examples/combined_ODNP.py

```diff
@@ -74,6 +76,10 @@
     raise ValueError("you said no!!!")
 powers = 1e-3 * 10 ** (dB_settings / 10.0)
 # }}}
+# {{{ these change if we change the way the data is saved
+IR_postproc = "spincore_IR_v3" # note that you have changed the way the data is saved, and so this should change likewise!!!!
```

Comment:
note this change

Reply:
I went to double check the processing with load_data and v1 is the one that accepts prechunked data (like here) but still FTs with a shift

Reply:
the change here seems to allow for the possibility that it's not phase cycled.  Does the postproc do that?

Reply:
no we need a PR in general to update postprocessing (at least the load_data.py/initial processing) but if we keep this as v3 (which doesn't exist) this can serve as a reminder?